ner_extended/server_setup.sh

```diff
@@ -0,0 +1,17 @@
+# Setup for initial training of models for Virtual Machine
```

Comment:
You have two setup files?

Reply:
Yes. The first one installs system upgrades on the server (and reboots, which makes it impossible to do it in 1 script).
The second one installs dependencies in a v_env.

Reply:
add a description of this somewhere probably in the readme